ocrd_tesserocr/ocrd-tool.json

```diff
@@ -202,7 +202,81 @@
         }
-      }
+      },
+      "resource_locations": ["data", "system", "module"],
+      "resources": [
+        {
+          "url": "https://ub-backup.bib.uni-mannheim.de/~stweil/ocrd-train/data/Fraktur_5000000/tessdata_fast/Fraktur_50000000.334_450937.traineddata",
+          "name": "Fraktur_GT4HistOCR.traineddata",
+          "parameter_usage": "without-extension",
+          "description": "Tesseract LSTM model trained on GT4HistOCR",
+          "size": 1058487
+        },
+        {
+          "url": "https://ub-backup.bib.uni-mannheim.de/~stweil/ocrd-train/data/ONB/tessdata_best/ONB_1.195_300718_989100.traineddata",
+          "name": "ONB.traineddata",
+          "parameter_usage": "without-extension",
+          "description": "Tesseract LSTM model based on Austrian National Library newspaper data",
+          "size": 4358948
+        },
+        {
+          "url": "https://github.com/tesseract-ocr/tessdata_fast/raw/main/equ.traineddata",
+          "name": "equ.traineddata",
+          "parameter_usage": "without-extension",
+          "description": "Tesseract equ model",
+          "size": 2251950
+        },
+        {
+          "url": "https://github.com/tesseract-ocr/tessdata_fast/raw/main/osd.traineddata",
+          "name": "osd.traineddata",
+          "parameter_usage": "without-extension",
+          "description": "Tesseract osd model",
+          "size": 10562727
+        },
+        {
+          "url": "https://github.com/tesseract-ocr/tessdata_fast/raw/main/eng.traineddata",
+          "name": "eng.traineddata",
+          "parameter_usage": "without-extension",
+          "description": "Tesseract eng model",
+          "size": 4113088
+        },
+        {
+          "url": "https://github.com/tesseract-ocr/tessdata_fast/raw/main/deu.traineddata",
+          "name": "deu.traineddata",
+          "parameter_usage": "without-extension",
+          "description": "Tesseract deu model",
+          "size": 1525436
+        },
+        {
+          "url": "https://github.com/tesseract-ocr/tessdata_fast/raw/main/frk.traineddata",
+          "name": "frk.traineddata",
+          "parameter_usage": "without-extension",
+          "description": "Tesseract frk model",
+          "size": 6423052
+        },
+        {
+          "url": "https://github.com/tesseract-ocr/tessdata_fast/raw/main/script/Fraktur.traineddata",
+          "name": "Fraktur.traineddata",
+          "parameter_usage": "without-extension",
+          "description": "Tesseract Fraktur model",
+          "size": 10915632
+        },
+        {
+          "url": "https://github.com/tesseract-ocr/tessdata_fast/raw/main/script/Latin.traineddata",
+          "name": "Latin.traineddata",
+          "parameter_usage": "without-extension",
+          "description": "Tesseract Latin model",
+          "size": 89384811
+        },
+        {
+          "url": "https://github.com/tesseract-ocr/tesseract/archive/main.tar.gz",
```

Comment:
Interesting – I did not know this exists.

But shouldn't we be able to expect any realistic installation of Tesseract to already contain this preinstalled in the module directory (i.e. under TESSDATA_PREFIX)?

Reply:
Probably. I just wanted to make sure that the configs are also installable via `resmgr`. If you don't think it's needed, we can drop it.

Reply:
Let's keep it for now (but in the future we should try to find a way to identify that archive's content with the config files that usually get installed by Tesseract).